core/lowering/lowering.cpp

```diff
@@ -35,7 +35,7 @@ void LowerGraph(std::shared_ptr<torch::jit::Graph>& g) {
     passes::Conv2DToConvolution(g);
     passes::FuseAddMMBranches(g);
     torch::jit::EliminateCommonSubexpression(g);
-    torch::jit::UnrollLoops(g);
+    //torch::jit::UnrollLoops(g);
```

Comment:
Does the unroll loops behavior still not work even with the non compliant converters?